docs: Style and formatting improvements to User Guide (#22622)
argoproj/argo-cd: docs/user-guide/multiple_sources.md

at that URL. If the `path` field is not set, Argo CD will use the repository solely as a source of value files.

!!! note

>     Sources with the `ref` field set must not also specify the `chart` field. Argo CD does not currently support using another Helm chart as a source for value files.

Reviewer: ```suggestion
    Sources with the `ref` field set cannot include the `chart` field. Currently, Argo CD does not support using another Helm chart as a source for value files.
```
